src/api/procedures/addTransferRestriction.ts

```diff
@@ -89,47 +118,94 @@ export async function prepareAddTransferRestriction(
     });
   }
 
-  const rawTransferManager = transferRestrictionToTransferManager({ type, value }, context);
+  const rawTransferCondition = transferRestrictionToPolymeshTransferCondition(
+    { type: chainType, value },
+    context
+  );
 
-  const transactions = [
-    checkTxType({
-      transaction: statistics.addTransferManager,
-      args: [rawTicker, rawTransferManager],
-    }),
-  ];
+  // BTreeSets need to be sorted
```

Comment:
`sort` will corece to string and sort by alphabetical order.

This should be stable, but there has to be a better way for this, since without it the error is `invalid signature` which is misleading if you don't know the cause.